ack.yaml

```diff
@@ -1,4 +1,3 @@
-# Generated from https://git.alpinelinux.org/aports/plain/main/ack/APKBUILD
```

Comment:
Comment removed

Reply:
I think it's ok, it was originally added by melange convert and most of the time we removed these